Automattic/sensei
# Add upgrade nudge to Quiz block settings

Fixes #4757

### Changes proposed in this Pull Request

* Add Quiz Timer promo setting to the Quiz block.
* Hide the _Number of Questions_ setting when _Random Question Order_ is disabled.

### Testing instructions

* Go to the lesson edit screen.
* Select the quiz block.
* Make sure the Quiz Timer setting is visible and the link is pointing to the price page.
* Enable the _Random Question Order_ setting and make sure the _Number of Questions_ is displayed.

### New Hooks

* `senseiQuizTimerHide` - Filters the quiz timer promo component display.

### Screenshot / Video

<img width="280" alt="Screenshot on 2022-02-15 at 03-50-59" src="https://user-images.githubusercontent.com/1612178/153977564-4c63f780-3de1-486b-b064-632e7807bb3d.png">
